app/common/components/kendo-grid/editable-cell/get-password-cell.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+
+import { Input } from '@servicetitan/design-system';
+
+import { DropDownListChangeEvent } from '@progress/kendo-react-dropdowns';
+
+import { getEditableCell, EditorProps } from './get-editable-cell';
+
+export function getPasswordCell() {
```

Comment:
You don’t need function in this case, just export component(result of getEditableCell calling)